### DISCUSSION TITLE
Different information for two heating circuits

### DISCUSSION THREAD

**Discussion by redfusion83** [emsesp/EMS-ESP32#2058] [Q&A] [answered]
There are two heating circuits in my system, each has its own mixer module. However, only one of them shows information like current flow temperature, as shown in the following screenshots. Does anyone know if and how I can get this information for both heating circuits?

My boiler is a Buderus GB192i.2, the mixer modules are both MM100.

<img width="1007" alt="Bildschirmfoto 2024-09-27 um 07 03 06" src="https://github.com/user-attachments/assets/969d249b-72aa-4a40-802f-f888e4f0b55a">

<img width="1009" alt="Bildschirmfoto 2024-09-27 um 07 03 23" src="https://github.com/user-attachments/assets/712d6a21-3143-46e9-b26a-e7c3f252e880">

**Comment by MichaelDvP** [comment]
There were some confusions about the telegrams for different heatingcircuits. Maybe the config telegram for hc1 is wrong. 
Please open a terminal and type `read 20 2D7` (for missing flow temp), `read 20 2CD` (actual hc1 config), `read 20 2CC` (possible config), `read 21 2CE` (hc2 config), `read 21 2D8` (hc2 data) 
Copy the result here.

**Comment by redfusion83** [comment]
Thanks for replying! Here are the results:

```
ems-esp:$ read 20 2D7
000+16:30:39.110 N 2: [emsesp] mixer(0x20) -W-> Me(0x0B), MMPLUSStatusMessage(0x02D7), data: 01 00 00 80 00 25 20 00 08

ems-esp:$ read 20 2CD
000+16:31:06.818 N 3: [emsesp] mixer(0x20) -W-> Me(0x0B), ?(0x02CD), data: 00 0E 05 FF 1E 00 00 00 FF

ems-esp:$ read 20 2CC
000+16:31:34.946 N 4: [emsesp] mixer(0x20) -W-> Me(0x0B), MMPLUSSetMessage(0x02CC), data: <empty>

ems-esp:$ read 21 2CE
000+16:32:09.361 N 5: [emsesp] mixer(0x21) -W-> Me(0x0B), MMPLUSSetMessage(0x02CE), data: FF 0E 05 FF 1E 00 00 00 FF

ems-esp:$ read 21 2D8
000+16:32:23.156 N 6: [emsesp] mixer(0x21) -W-> Me(0x0B), MMPLUSStatusMessage(0x02D8), data: 01 00 1E 01 2F 1D 20 00 08
```

**Reply by MichaelDvP** [reply, marked as answer]
Oh, seems you are using 3.6.5 software. Update to the 3.7.0dev version and it will work.

BTW: Your hc1 mixer have no temperaturesensor installed (code 8000 in offset 3 of 0x02D7), so you will not get the flowtemp.

**Reply by redfusion83** [reply]
Thanks, that worked. With 3.7.0 I have the same information/settings for both mixers minus the hc1 flow temperature.

Just out of curiosity, where would such a thermostat be installed like it is on hc2? I'm just asking myself why there is only one.

**Reply by MichaelDvP** [reply]
The sensor is placed on the outflow of the mixing unit (near the red temperature indicator). Could be that there is a sensor connected to the boiler (flowtemp) and the mixer takes the value from there over the bus to control the valve.
